src/NuGet.Core/NuGet.Build.Tasks.Pack.Library/Pack.targets

```diff
@@ -56,6 +56,16 @@ Copyright (c) .NET Foundation. All rights reserved.
           DependsOnTargets="Pack">
   </Target>
 
+  <Target Name="_CleanPackageFiles"
+          AfterTargets="Clean"
```

Comment:
One of the commonly recommended ways to extend cleaning process is to override `CleanDependsOn` property:
```xml
<CleanDependsOn> CustomBeforeClean; $(CleanDependsOn); CustomAfterClean </CleanDependsOn>
```
[here](https://msdn.microsoft.com/en-us/magazine/dd419659.aspx) and [here](https://msdn.microsoft.com/en-us/library/ms366724.aspx).

Do you think we should prefer the other alternative?